app/dao/service_callback_api_dao.py

```diff
@@ -32,6 +32,10 @@ def get_service_callback_api(service_callback_api_id, service_id, callback_type)
     ).first()
 
 
+def get_service_callback_api_by_callback_type(service_id, callback_type):
```

Comment:
do we need both `get_service_callback_api` and `get_service_callback_api_by_callback_type`? They are very similar.

Reply:
It is a temporary addition just to solve the issue where an update is being applied to an inbound_sms callback which has been written to both tables. The callback will be have different ids in both tables, and because we are fetching the data from the `service_inbound_api` table, the id presented to the update_service_callback_api endpoint would be that of the row in the `service_inbound_api` table. This new dao method helps us to retrieve an `inbound_sms` callback present for the service by callback type. It would be removed once the table merge is completed.